aws_marketplace/using_model_packages/amazon_augmented_ai_with_aws_marketplace_ml_models/amazon_augmented_ai_with_aws_marketplace_ml_models.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<p>Suggestions:</p><ul><li>Bold the things that users have to click on.</li><li>Users must add workers on the same screen they use to enter the team name when creating the team for the first time</li><li>Add a note that users are recommended to use "Create a private team with AWS Cognito".</li></ul><p>Here's a new version:</p><p> </p><p>If you have already created a private work team, replace <code>&lt;WORKTEAM_ARN&gt;</code> with the ARN of your work team. If you have never created a private work team, use the instructions below to create one. To learn more about using and managing private work teams, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-workforce-private.html)" target="_blank">Use a Private Workforce</a>.</p><p> </p><ol><li>In the Amazon SageMaker console in the left sidebar under the Ground Truth heading, open the <strong>Labeling Workforces</strong>.</li><li>Choose <strong>Private</strong>, and then choose <strong>Create private team</strong>.</li><li>If you are a new user of SageMaker workforces, it is recommended you select <strong>Create a private work team with AWS Cognito.</strong></li><li>For team name, enter "MyTeam".</li><li>To add workers by email, select <strong>Invite new workers by email</strong> and paste or type a list of up to 50 email addresses, separated by commas, into the email addresses box. If you are following this blog post, specify an email account that you have access to. The system sends an invitation email, which allows users to authenticate and set up their profile for performing human-in-loop review.</li><li>Enter an organization name - this will be used to customize emails sent to your workers.</li><li>For contact email, enter an email address you have access to.</li><li>Select <strong>Create private team</strong>.</li></ol><p>This will bring you back to the Private tab under labeling workforces, where you can view and manager your private teams and workers.</p><p> </p><h3><strong>IMPORTANT: After you have created your work team, from the Private teams section copy the value of the ARN and uncomment and replace </strong><code><strong>&lt;WORKTEAM_ARN&gt;</strong></code><strong> below:</strong></h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1980/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I incorporated these suggestions. Thanks.